actors/market/src/lib.rs

```diff
@@ -1135,11 +1133,7 @@ where
         }
     }
 
-    Ok(DealWeights {
-        deal_space: total_deal_size,
-        deal_weight: total_deal_space_time,
-        verified_deal_weight: total_verified_space_time,
-    })
+    Ok(DealSizes { deal_space: deal_size, verified_deal_space: verified_deal_size })
```

Comment:
This changes the semantics of `deal_space` from the total to only the unverified part. However, the value appears to be unused so this is ok. Perhaps comment on the struct that it's only the unverified part.